

Invoking dbt with ['seed', '--select', 'seed', '--full-refresh']
[0m10:28:32  Running with dbt=1.7.18
[0m10:28:32  Registered adapter: firebolt=1.6.4
[0m10:28:32  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 431 macros, 0 groups, 0 semantic models
[0m10:28:32  
[0m10:28:32  Concurrency: 1 threads (target='default')
[0m10:28:32  
[0m10:28:32  1 of 1 START seed file seed .................................................... [RUN]
[0m10:28:33  1 of 1 OK loaded seed file public.seed ......................................... [[32mCREATE 6[0m in 0.84s]
[0m10:28:33  
[0m10:28:33  Finished running 1 seed in 0 hours 0 minutes and 1.44 seconds (1.44s).
[0m10:28:33  
[0m10:28:33  [32mCompleted successfully[0m
[0m10:28:33  
[0m10:28:33  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1


Invoking dbt with ['run', '--select', 'not_found_unique_key', '--full-refresh']
[0m10:28:33  Running with dbt=1.7.18
[0m10:28:33  Registered adapter: firebolt=1.6.4
[0m10:28:33  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 431 macros, 0 groups, 0 semantic models
[0m10:28:33  
[0m10:28:34  Concurrency: 1 threads (target='default')
[0m10:28:34  
[0m10:28:34  1 of 1 START sql incremental model not_found_unique_key ........................ [RUN]
[0m10:28:35  1 of 1 OK created sql incremental model not_found_unique_key ................... [[32mSUCCESS[0m in 0.84s]
[0m10:28:35  
[0m10:28:35  Finished running 1 incremental model in 0 hours 0 minutes and 1.44 seconds (1.44s).
[0m10:28:35  
[0m10:28:35  [32mCompleted successfully[0m
[0m10:28:35  
[0m10:28:35  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1


Invoking dbt with ['run', '--select', 'not_found_unique_key']
[0m10:28:35  Running with dbt=1.7.18
[0m10:28:35  Registered adapter: firebolt=1.6.4
[0m10:28:35  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 431 macros, 0 groups, 0 semantic models
[0m10:28:35  
[0m10:28:36  Concurrency: 1 threads (target='default')
[0m10:28:36  
[0m10:28:36  1 of 1 START sql incremental model not_found_unique_key ........................ [RUN]
[0m10:28:36  1 of 1 ERROR creating sql incremental model not_found_unique_key ............... [[31mERROR[0m in 0.72s]
[0m10:28:36  
[0m10:28:36  Finished running 1 incremental model in 0 hours 0 minutes and 1.36 seconds (1.36s).
[0m10:28:36  
[0m10:28:36  [31mCompleted with 1 error and 0 warnings:[0m
[0m10:28:36  
[0m10:28:36    Runtime Error in model not_found_unique_key (models/not_found_unique_key.sql)
  Error executing query:
  {
    "errors": [
      {
        "description": "Line 3, Column 17: Column 'thisisnotacolumn' does not exist."
      }
    ],
    "query": {
      "query_id": "60c9fb7e-0ed2-4ff4-a353-0efdb27596c4",
      "request_id": "d7ba92c9-df07-4de7-8125-6adbb61402a6"
    },
    "statistics": {
      "elapsed": 0.0
    }
  }
  
[0m10:28:36  
[0m10:28:36  Done. PASS=0 WARN=0 ERROR=1 SKIP=0 TOTAL=1
